components/pango_packetstream/include/pangolin/log/packetstream_tags.h

```diff
@@ -2,6 +2,7 @@
 
 #include <cstdint>
 #include <string>
+#include <stdint.h>
```

Comment:
The error message suggests `#include <cstdint>` (`note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?`) But this is already included. Could this be an issue with the compiler?